spec/shoryuken/worker_spec.rb

```diff
@@ -99,36 +100,64 @@
   end
 
   describe '.shoryuken_options' do
-    it 'registers a worker' do
-      expect(Shoryuken.worker_registry.workers('default')).to eq([TestWorker])
+    let(:dummy_worker) { Class.new }
+
+    before do
+      dummy_worker.include Shoryuken::Worker
     end
 
-    it 'accepts a block as queue name' do
-      $queue_prefix = 'production'
+    subject { dummy_worker.shoryuken_options shoryuken_options }
+
+    context "when using the queue key" do
+      let(:shoryuken_options) {{ "queue" => "a_queue" }}
 
-      class NewTestWorker
-        include Shoryuken::Worker
+      it "warns about deprecation" do
+        expect(Shoryuken.logger).to receive(:warn).
+          with("[DEPRECATION] queue is deprecated as an option in favor of multiple queue support, please use queues instead").
+          once
+        subject
+      end
+
+      it "does not keep the value from queue" do
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.